.github/pull_request_template.md

```diff
@@ -0,0 +1,33 @@
+# Description
+
+Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.
+
+Fixes # (issue)
+
+## Type of change
+
+Please delete options that are not relevant.
+
+- [ ] Bug fix (non-breaking change which fixes an issue)
+- [ ] New feature (non-breaking change which adds functionality)
+- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
+- [ ] This change requires a documentation update
+
+# How Has This Been Tested?
+
+Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. 
+
+- [ ] Test A
+- [ ] Test B
+
+
+# Checklist:
```

Comment:
Maybe reduce the checklist and link to the contributing.md...

Reply:
I am not sure. 
On one hand it will remove the duplication of the text. 
On the other hand this makes it simple to follow for the developer creating the PR

Reply:
I'm for keeping this as well

Reply:
Added link to the Contributing